### ISSUE TITLE
Is this project still supported?

### ISSUE THREAD

**Issue by SebastianOsinski** [graphhopper/graphhopper-ios#40]
Hi! I see lots of traffic and updates on main repo, but this repo was last updated in 2017. Are there any plans to update iOS version to 0.13?

**Comment by karussell** [comment]
The maintainer was @clns. If someone else has interests to take over maintaining this project then we would appreciate and support it. But we do not have iOS experience to push this forward.

**Comment by clns** [comment]
Yes, unfortunately I can't maintain this repo at this moment, but I'm glad to help with any questions.

**Comment by SebastianOsinski** [comment]
Thank you for quick responses!

I had some problems building the project, but after some digging through issues I managed to do that:

1. Instead of building library with Xcode (which gave me error described in #39), I built it directly with `make` as suggested here https://github.com/graphhopper/graphhopper-ios/issues/39#issuecomment-542568958
2. Sample app now builds, but I keep getting Runtime error. Same as described in #31. All translations are in project. Is it possible that library can't access files in NSBundle?

**Comment by songyuyang0918** [comment]
@SebastianOsinski  Is Xcode working perfectly now?

**Comment by SebastianOsinski** [comment]
@songyuyang0918 I haven’t tried it since my last comment as we decided to postpone this feature in our project

**Comment by innodeltech** [comment]
@clns 

We are trying to use graphhopper-ios and the version in GIT is 0.8.0 which very old. is there a new version which you can provide please? Since we have been able to use it successfully on Android we want to use the same on iOS also. 

We could make a successful build but "libgraphhopper.a" file is not getting created and the program doesn't run on iOS phone but runs within the emulator. how can we fix this libgraphhopper.a file not getting created and make this run smooth on ios phone. 

We will appreciate any advice/direction/instructions to fix this issue. This is a bit urgent if you can please help.

**Comment by clns** [comment]
This project hasn't been updated to the latest versions of graphhopper unfortunately. I cannot maintain this project, so if anyone else wants to update it, that will be welcomed.
